django_cloud_deploy/skeleton/templates/settings_template/base_settings.py-tpl

```diff
@@ -56,7 +56,21 @@ MIDDLEWARE = [
 
 ROOT_URLCONF = '{{ project_name }}.urls'
 
-TEMPLATES = []
+TEMPLATES = [
```

Comment:
In settings file generated by "django-admin startproject", "TEMPLATES" is equal to this value.